tests/regression_tests/source_mcpl_file/gen_dummy_mcpl.c

```diff
@@ -0,0 +1,98 @@
+#include <mcpl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+
+const int batches=10;
+const int particles=10000;
+
+double rand01(){
+  double r=rand()/((double) RAND_MAX);
+  return r;
+}
+
+
+int main(int argc, char **argv){
```

Comment:
Instead of having this .c file that needs to be compiled, you could just include a .mcpl file in the repo directly (as long as the size is small).

Reply:
The c-file is not needed anymore, as it is now generated by test itself. OTOH I _did_ forget to remove it, so it was good that you spotted it.